data-raw/logos.R

```diff
@@ -47,6 +47,99 @@ logos <- leagues %>%
 
 
 
+## Renaming ncaa team color names, so that they match up with the join
+## This is not the best way to do this, but it was important to keep the mascots in the team names
+
+## Old name from teamcolors dataset                                      <- New name to match logos dataset
+
+teamcolors$name[teamcolors$name == "Air Force"]                          <- "Air Force Falcons"
```

Comment:
Wow, @pep93 thank you for this Herculean effort! Here is what I would prefer:

1. Make this list into a two-column `data.frame`: the first column is `teamcolors_name` and the second column is `sportslogos_name`. You can do this right here in the code with `tribble()`.
2. Use `left_join()` to add the `sportslogos_name` to the existing `teamcolors` data frame. 

It's essentially the same logic as what you have here and roughly the same number of lines of code, but it will be easier to maintain.

Reply:
Just need to need to figure out the last mutate. Was traveling today. Will finish it again in the morning and commit the changes. I will also figure out what's going on with the R CMD check. Maybe I accidentally moved a file or something? Will check it out in the morning. Additionally, if there's any other feedback on the corrections I made today, just let me know! With a fresh pair of eyes, I can also see if there's any other tricks that could be used to further reduce the nasty gsub function... but it better than before.